linux_os/guide/system/software/integrity/crypto/harden_sshd_crypto_policy/rule.yml

```diff
@@ -0,0 +1,28 @@
+documentation_complete: true
+
+title: 'Harden SSHD Crypto Policy'
+
+description: |-
+    Crypto Policies are means of enforcing certain cryptographic settings for selected applications including OpenSSH server.
+    The SSHD service is by default configured to modify its configuration based on currently configured Crypto-Policy. However, in certain cases it might be needed to override the Crypto Policy specific to OpenSSH Server and leave rest of the Crypto Policy in tact.
```

Comment:
Typo:`intact`

Reply:
Suggestion for text: `However, it may be necessary to customize the Crypto Policy only for OpenSSH Server.`